atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -156,6 +173,21 @@ public static SerializableTransactionManager create(
         return create(config, schemas, env, lockServerOptions, allowHiddenTableAccess, UserAgents.DEFAULT_USER_AGENT);
     }
 
+    /**
+     * Create a {@link SerializableTransactionManager} with provided configuration, a set of
+     * {@link Schema}s, {@link LockServerOptions}, and an environment in which to register HTTP server endpoints.
+     */
+    public static SerializableTransactionManager create(
```

Comment:
The chain of various create() methods is now almost impossible to follow without an IDE :)

Do I understand correctly that this change doesn't actually implement live reloading and we still use a constant supplier?